paxml/train.py

```diff
@@ -45,6 +45,7 @@
 from praxis import base_layer
 from praxis import py_utils
 from praxis import pytypes
+from statistics import mean
```

Comment:
lets use numpy or jax.numpy for mean? we don't typically use python statistics module